### PR TITLE
Makes bags of holding great again

### PR DESCRIPTION
**Pull request by ghost**
:cl: steamp0rt
add: Bags of holding now make singulos again
/:cl:

### CODE REVIEW

**Comment by WarRage333** [comment]
Bad

**Comment by Manly-Dwarf** [comment]
I'm not a fan of the singulo change either, but a better alternative would be to just buff the tear in reality. Make it much bigger.
You also just straight up unincluded the whole file, so...

**Comment by ghost** [comment]
>  You also just straight up unincluded the whole file, so...

Because sometimes using `...()` makes warnings

**Comment by MrWankman** [comment]
I mean I suppose whether you prefer singulo to tear is dependent on whether you think bag of holding should either just be an obstical in the station or end the round. I'm fine with it making a tear myself tho.

**Comment by ExpAntique** [comment]
-1

File: hippiestation.dme [hunk 1]
```diff
@@ -389,7 +389,6 @@
 #include "code\datums\components\decals\blood.dm"
 #include "code\datums\components\storage\storage.dm"
 #include "code\datums\components\storage\concrete\_concrete.dm"
-#include "code\datums\components\storage\concrete\bag_of_holding.dm"
```
Comment (Jujumatic) [inline hippiestation.dme:392]: we still need this included in our tg code, the hippiestation code should overwrite it anyway but just makes it easier for us to keep original tg code and keep it updated

**Review by Jujumatic — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
terrible idea

File: hippiestation.dme (hunk 1 above)
Comment (ghost) [reply hippiestation.dme:392]: no, even the old code calls `..()`

**Comment by Lxtoofonea** [comment]
No, that was way too OP in the past, but I agree with kbb on one, just make the tear worse

**Comment by ghost** [comment]
tear is dumb tho

**Comment by JohnMcLaime** [comment]
Tear in reality is already strong enough, i would say it is better than singularity which was easy to escape compared to this. When someone boh bombed, singu would appear in smallest form, easy to run away from and if someone was boh bombing with auto cloning, it had potential to fuck them over, unless they invested extra time in making autocloner off station zlvl. This current one is retarded, just upgrade cloner and boh bomb spam, everyone in screen range gets instafucked, if they dont get gibbed by tear, they will get stuck in all the clutter or will die to space since ladders lead there iirc, unless they HAPPEN to get lucky with tear rng and have hand tele locked on station beacon or hardsuit with enough oxy for a trip to station.

**Comment by JohnMcLaime** [comment]
Revert this tg piece of shit, they wanted to nerf it with this change iirc, but they actually buffed it, fuck their monkey coders.

**Comment by yskefunky** [comment]
what if as a middle ground you'd have to do some shit to the bag of holding to shit out sings? emag or something? otherwise nah -1

**Comment by danhibiki** [comment]
-2 headmins this has been voted out

**Comment by earthmoonjune** [comment]
-1

**Comment by GuyonBroadway** [comment]
Two headmins have denied this, closing until you convince them otherwise.

**Pull request closed without merging**